.maintain/push-image.sh

```diff
@@ -6,24 +6,25 @@ target_tag=$2
 PROVIDER_BIN_NAME="dip-provider-node-template"
 CONSUMER_BIN_NAME="dip-consumer-node-template"
 
-# publish to docker hub
-docker tag local/standalone-node:$source_tag ${DOCKER_HUB_STANDALONE}:$target_tag
-docker tag local/kilt-node:$source_tag ${DOCKER_HUB_PARACHAIN}:$target_tag
-docker tag local/$PROVIDER_BIN_NAME:$source_tag ${DOCKER_HUB_DIP_PROVIDER_TEMPLATE}:$target_tag
-docker tag local/$CONSUMER_BIN_NAME:$source_tag ${DOCKER_HUB_DIP_CONSUMER_TEMPLATE}:$target_tag
+# Function to tag and push images
+tag_and_push() {
+  local source=$1
+  local target=$2
 
-docker push ${DOCKER_HUB_STANDALONE}:$target_tag
-docker push ${DOCKER_HUB_PARACHAIN}:$target_tag
-docker push ${DOCKER_HUB_DIP_PROVIDER_TEMPLATE}:$target_tag
-docker push ${DOCKER_HUB_DIP_CONSUMER_TEMPLATE}:$target_tag
+  docker tag $source $target
+  docker push $target &
+}
 
-# publish to AWS
-docker tag local/standalone-node:$source_tag $AWS_REGISTRY/kilt/prototype-chain:$target_tag
-docker tag local/kilt-node:$source_tag $AWS_REGISTRY/kilt-parachain/collator:$target_tag
-docker tag local/$PROVIDER_BIN_NAME:$source_tag $AWS_REGISTRY/$PROVIDER_BIN_NAME:$target_tag
-docker tag local/$CONSUMER_BIN_NAME:$source_tag $AWS_REGISTRY/$CONSUMER_BIN_NAME:$target_tag
+# Tag and push to Docker Hub
+tag_and_push local/standalone-node:$source_tag ${DOCKER_HUB_STANDALONE}:$target_tag
+tag_and_push local/kilt-node:$source_tag ${DOCKER_HUB_PARACHAIN}:$target_tag
+tag_and_push local/$PROVIDER_BIN_NAME:$source_tag ${DOCKER_HUB_DIP_PROVIDER_TEMPLATE}:$target_tag
+tag_and_push local/$CONSUMER_BIN_NAME:$source_tag ${DOCKER_HUB_DIP_CONSUMER_TEMPLATE}:$target_tag
 
-docker push $AWS_REGISTRY/kilt/prototype-chain:$target_tag
-docker push $AWS_REGISTRY/kilt-parachain/collator:$target_tag
-docker push $AWS_REGISTRY/$PROVIDER_BIN_NAME:$target_tag
-docker push $AWS_REGISTRY/$CONSUMER_BIN_NAME:$target_tag
+# Tag and push to AWS
+tag_and_push local/standalone-node:$source_tag $AWS_REGISTRY/standalone-node:$target_tag
```

Comment:
NB: I created `kilt-node` and `standalone-node` private repos to mirror what we got on docker.io & AWS